pom.xml

```diff
@@ -768,6 +768,22 @@
                 <version>${dep.okhttp.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.jsonwebtoken</groupId>
+                <artifactId>jjwt</artifactId>
+                <version>0.9.0</version>
+                <exclusions>
+                    <exclusion>
```

Comment:
Might be better to add an exception for `maven-dependency-versions-check-plugin` below, as these are actually required but the version is wrong